lib/workers/repository/process/vulnerabilities.ts

```diff
@@ -454,12 +454,16 @@ export class Vulnerabilities {
     logger.debug(
       `Setting allowed version ${fixedVersion} to fix vulnerability ${vulnerability.id} in ${packageName} ${depVersion}`
     );
+
+    const severityLevel = vulnerability.database_specific?.severity as string;
```

Comment:
Rather than `database_specific` which changes based source, use the general `vulnerability.severity`

Reply:
I chose `vulnerability.database_specific?.severity` so that it was consistent with what is used within the prBody https://github.com/renovatebot/renovate/blob/c34ebaa0ea1650c1c8fcc4ba20f78b8b25cfa130/lib/workers/repository/process/vulnerabilities.ts#L533-L537

I did look into `vulnerability.severity`, but it doesn't look suitable.  It is sometimes not present at all (ie: when testing `express-cart` vuls), or when it is (ie: testing `karma` vuls) it is an array containing a score and type (no severity)

Reply:
This works only for GitHub security advisories that provide no CVSS base score and is meant as a last resort / fallback instead of showing "Unknown".

Reply:
@Churro - ah ok, got it.  i'll rework the logic

Reply:
Logic extracted to `extractSeverityDetails` so it can be reused.